gtk4/src/auto/bookmark_list.rs

```diff
@@ -61,6 +64,20 @@ impl BookmarkList {
         }
     }
 
+    #[cfg(any(feature = "v4_8", feature = "dox"))]
+    #[cfg_attr(feature = "dox", doc(cfg(feature = "v4_8")))]
+    #[doc(alias = "item-type")]
+    pub fn item_type(&self) -> glib::types::Type {
```

Comment:
I'm not sure you want methods here calling into the properties; the `gio::ListModel` API requires a cast, but it's still going to be faster than involving `g_object_get_property()` and `GValue`…

Reply:
Erm why do we create property accessors for properties defined by interfaces? That seems like a bug to me :) All these are already available via the `ListModel` API and just a duplication like this.

Can you create a gir issue for this?

Reply:
Created https://github.com/gtk-rs/gir/issues/1369